server/src/main/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdMonitor.java

```diff
@@ -140,6 +140,10 @@ public void onNewInfo(ClusterInfo info) {
             final DiskUsage usage = entry.value;
             final RoutingNode routingNode = routingNodes.node(node);
 
+            final long reservedSpace = info.getReservedSpace(usage.getNodeId(), usage.getPath()).getTotal();
+            final DiskUsage usageWithReservedSpace = new DiskUsage(usage.getNodeId(), usage.getNodeName(), usage.getPath(),
```

Comment:
Can you move the creation of this object to the place where it's first used? Or should we  alternatively call the other `usage` instead `usageWithoutReservedSpace` to make it clearer  when we are using one vs the other

Reply:
I tried renaming it but I found that harder to distinguish (`usageWithoutReservedSpace` and `usageWithReservedSpace` are pretty similar to the eye). Moved the construction lower down in 56ecdb71c243a9fdac03826d8c5b210ccb18d23b.